src/runtime/task/mod.rs

```diff
@@ -464,14 +539,13 @@ pub(crate) struct ParkState {
 pub struct TaskId(pub(super) usize);
 
 impl Debug for TaskId {
+    // If the `TaskName` label is set, use that when generating the Debug string
```

Comment:
Might be nice to make `std::thread::Builder` set the `TaskName` label on a new thread if it has a `.name()` set?

Reply:
Think I'm already doing that: `Builder::spawn` calls `thread::spawn_named` which calls `ExecutionState::spawn_thread` which calls `ExecutionState::set_labels_for_new_task` which sets the name if one was provided.

Reply:
I added a check to `tests::basic::thread::thread_builder_name` to check that the name provided to the builder is assigned when you call `get_name_for_task()`.